Demo/Program.cs

```diff
@@ -1,26 +1,108 @@
-﻿using External.Encoder;
-using External.Notification;
+﻿using Encoder;
+using Notification;
 using System;
+using System.Text;
 
 namespace Demo
 {
     class Program
     {
+        private static INotificationProvider _notificationProvider;
         static void Main(string[] args)
         {
-            var videoEncoder = new VideoEncoder();
-            var audioEncoder = new AudioEncoder();
-
-            var emailService = new EmailService();
             var smsService = new SmsService();
+            var emailService = new EmailService();
+            var telegramService = new TelegramService();
+            _notificationProvider = new NotificationProvider();
+
+            _notificationProvider.Add(emailService);
+            _notificationProvider.Add(smsService);
+            _notificationProvider.Add(telegramService);
+
+            Console.WriteLine(SourceUploadMessage());
+
+            var sourceType = Console.ReadKey();
+            Console.WriteLine(Environment.NewLine);
```

Comment:
There is no need to add Environment.NewLine. WriteLine already has this logic in it.